src/ERC20Permit.sol

```diff
@@ -40,9 +48,11 @@ abstract contract ERC20Permit is ERC20 {
         bytes32 r,
         bytes32 s
     ) external {
-        require(block.timestamp < deadline, 'EXPIRED');
-        require(uint256(s) <= 0x7FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF5D576E7357A4501DDFE92F46681B20A0, 'INV_S');
-        require(v == 27 || v == 28, 'INV_V');
+        if (block.timestamp > deadline) revert PermitExpired();
```

Comment:
Now `timestamp = deadline` is considered valid, is that intentional?

Reply:
Yes. It's actually how it should have been.